### PR TITLE
Telemetry data collection updates

### PR DESCRIPTION
**Pull request by mikefrobbins**
Updated telemetry data information as requested.

### CODE REVIEW

File: azps-3.8.0/Az.Accounts/Disable-AzDataCollection.md [hunk 1]
```diff
@@ -20,27 +20,30 @@ Disable-AzDataCollection [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-
...
 
 ## EXAMPLES
 
 ### Example 1: Disabling data collection for the current user
-```
+
+The following example shows how to disable data collection for the current user.
+
+```powershell
 PS C:\> Disable-AzDataCollection
```
Comment (sdwheeler) [inline azps-3.8.0/Az.Accounts/Disable-AzDataCollection.md:39]: Consider removing prompts from examples.

File: docs-conceptual/azps-3.8.0/get-started-azureps.md [hunk 2]
```diff
@@ -6,76 +6,105 @@ ms.author: sttramer
...
 Connect-AzAccount -Environment AzureChinaCloud
 ```
 
-In PowerShell 5.1 environments, you'll get a sign-in dialog to provide a username and password for your Azure
-account. On every other version of PowerShell, you'll get a token to use on [https://microsoft.com/devicelogin].
-Open this page in your browser and enter the token, then sign in with your Azure account credentials and
-authorize Azure PowerShell.
+In Windows PowerShell 5.1 environments, you'll receive a sign-in dialog to provide a username and
+password for your Azure account. On every other version of PowerShell, you'll get a token to use on
+[https://microsoft.com/devicelogin]. Open this page in your browser and enter the token, then sign
```
Comment (sdwheeler) [inline docs-conceptual/azps-3.8.0/get-started-azureps.md:53]: Need to fix this bare url.

File: docs-conceptual/azps-3.8.0/get-started-azureps.md [hunk 3]
```diff
@@ -6,76 +6,105 @@ ms.author: sttramer
...
+Azure PowerShell cmdlets follow a standard naming convention for PowerShell, `Verb-Noun`. The verb
+describes the action (examples include `New`, `Get`, `Set`, `Remove`) and the noun describes the
+resource type (examples include `AzVM`, `AzKeyVaultCertificate`, `AzFirewall`,
+`AzVirtualNetworkGateway`). Nouns in Azure PowerShell always start with the prefix `Az`. For the
+full list of standard verbs, see
+[Approved verbs for PowerShell Commands](/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands).
 
-Knowing the nouns, verbs, and the Azure PowerShell modules available help you find commands with the [Get-Command](/powershell/module/microsoft.powershell.core/get-command) cmdlet. For example, to find all VM-related commands that use the `Get` verb:
+Knowing the nouns, verbs, and the Azure PowerShell modules available help you find commands with the
```
Comment (sdwheeler) [inline docs-conceptual/azps-3.8.0/get-started-azureps.md:77]:
```suggestion
Knowing the nouns, verbs, and the Azure PowerShell modules available helps you find commands with the
```

File: azps-3.8.0/Az.Accounts/Disable-AzDataCollection.md (hunk 1 above)
Comment (mikefrobbins) [reply azps-3.8.0/Az.Accounts/Disable-AzDataCollection.md]: Prompt has been removed.

File: docs-conceptual/azps-3.8.0/get-started-azureps.md (hunk 2 above)
Comment (mikefrobbins) [reply docs-conceptual/azps-3.8.0/get-started-azureps.md]: Fixed bare URL.

File: docs-conceptual/azps-3.8.0/get-started-azureps.md (hunk 3 above)
Comment (mikefrobbins) [reply docs-conceptual/azps-3.8.0/get-started-azureps.md]: Changed help to helps as suggested.

File: docs-conceptual/azps-3.8.0/get-started-azureps.md [hunk 4]
```diff
@@ -6,76 +6,105 @@ ms.author: sttramer
...
+use Cloud Shell. Your Azure Cloud Shell session is already authenticated for the environment,
+subscription, and tenant that launched the Cloud Shell session.
 
 ```azurepowershell-interactive
 Connect-AzAccount
 ```
 
-If you're in a non-US region, use the `-Environment` parameter to sign in. Get the name of the environment for your region by using
-the [Get-AzEnvironment](/powershell/module/Az.Accounts/Get-AzEnvironment) cmdlet. For example, to sign in to Azure China 21Vianet:
+If you're in a non-US region, use the **Environment** parameter to sign in. Get the name of the
```
Comment (dcaro) [inline docs-conceptual/azps-3.8.0/get-started-azureps.md:42]: I would not use "if you're in a non-US region", the environment is made to target the cloud this is not Azure Public. What about "If you need to use a national cloud, use ..." ?

**Comment by mikefrobbins** [comment]
#sign-off

**Pull request merged**